src/modules/commander/Commander.cpp

```diff
@@ -1766,7 +1766,7 @@ Commander::run()
 
 				// revert to position control in any case
 				main_state_transition(status, commander_state_s::MAIN_STATE_POSCTL, status_flags, &_internal_state);
-				mavlink_log_info(&mavlink_log_pub, "Autonomy off! Returned control to pilot");
+				mavlink_log_info(&mavlink_log_pub, "Pilot took over control by sticks");
```

Comment:
```suggestion
				mavlink_log_info(&mavlink_log_pub, "Pilot took over control using sticks");
```
?

Reply:
What's the maximum character number? It will probably work, I'll try. 👍

Reply:
![image](https://user-images.githubusercontent.com/4668506/74950031-d515dd80-53fe-11ea-90cc-73e5b28cd1b1.png)